### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by iamabhi222**
#646  
updated logo in readme.md

### CODE REVIEW

File: packages/vocabulary/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <a href="https://creativecommons.github.io/vocabulary/" class="readme-vocabulary-logo">
     <img
       alt="Vocabulary logo"
-      src="https://raw.githubusercontent.com/creativecommons/vocabulary/master/readme_assets/vocabulary_logo.svg?sanitize=true"
+      src="https://raw.githubusercontent.com/creativecommons/vocabulary/master/packages/vocabulary/readme_assets/vocabulary_logo.svg?sanitize=true"
```
Comment (dhruvkb) [inline packages/vocabulary/README.md]: This is not a full fledged solution.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
A proper solution would involve pulling the `readme_assets` folder out into the root directory so that all references to the directory are automatically fixed. That would also fix our `npm` page cc: @nimishbongale and @zackkrida.

**Comment by nimishbongale** [comment]
@dhruvkb I'm not sure if `readme_assets/cc_logo.png` is required here? The main `README.md` makes use only of `readme_assets/vocabulary_logo.svg` if I'm not mistaken.

We may further need to dissolve the `readme_assets` of all our packages.

**Comment by iamabhi222** [comment]
@nimishbongale yes if you move the _readme_assets_ to the root then the main readme.md link also must be changed.

**Comment by zackkrida** [comment]
@iamabhi222 thanks a ton for looking at this and discussing with the other contributors! I am going to close this though, because we're reworking things a little in #649 

I hope you contribute again, and soon 😄

**Pull request closed without merging**